lldb/source/Plugins/ObjectFile/ELF/ObjectFileELF.cpp

```diff
@@ -1477,32 +1506,32 @@ size_t ObjectFileELF::GetSectionHeaderInfo(SectionHeaderColl &section_headers,
   }
   if (idx < section_headers.size())
     section_headers.resize(idx);
+  // Sometimes we are able to read the section header memory from an in memory
+  // ELF file, but all section header data has been set to zeroes. Remove any
+  // SHT_NULL sections if we have more than 1. The first entry in the section
+  // headers should always be a SHT_NULL section, but none of the others should
+  // be.
+  if (section_headers.size() > 1 && section_headers[1].sh_type == SHT_NULL)
```

Comment:
Is it ever possible for section headers to be null? Should we validate it is `SHT_NULL` before erasure?